packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -1040,79 +1060,89 @@ export function TransferPanel() {
     [isSmartContractWallet, isDepositMode, gasSummary]
   )
 
-  const getErrorMessage = useCallback(
-    (
-      _amountEntered: string,
-      _balance: string | null
-    ): TransferPanelMainErrorMessage | undefined => {
-      // No error while loading balance
-      if (_balance === null || ethBalance === null) {
-        return undefined
-      }
+  const transferPanelMainErrorMessage:
```

Comment:
This used to be a `useCallback` that accepted either `l1Balance` or `l1Balance` to check against but it was refactored into a `useMemo`.